plugins/aws/ec2/unusedEni.js

```diff
@@ -0,0 +1,58 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Unused Elastic Network Interfaces',
+    category: 'EC2',
+    description: 'Ensures that unused AWS Elastic Network Interfaces (ENIs) are removed.',
+    more_info: 'Unused AWS ENIs should be removed to follow best practices and to avoid reaching the service limit.',
+    recommended_action: 'Delete the unused AWS Elastic Network Interfaces',
+    link: 'https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html',
+    apis: ['EC2:describeNetworkInterfaces', 'STS:getCallerIdentity'],
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        var regions = helpers.regions(settings);
+
+        var acctRegion = helpers.defaultRegion(settings);
+        var awsOrGov = helpers.defaultPartition(settings);
+        var accountId = helpers.addSource(cache, source, ['sts', 'getCallerIdentity', acctRegion, 'data']);
+
+        async.each(regions.ec2, function(region, rcb) {
+            var describeNetworkInterfaces = helpers.addSource(cache, source,
+                ['ec2', 'describeNetworkInterfaces', region]);
+
+            if (!describeNetworkInterfaces) return rcb();
+
+            if(describeNetworkInterfaces.err || !describeNetworkInterfaces.data) {
+                helpers.addResult(results, 3,
+                    `Unable to query AWS ENIs: ${helpers.addError(describeNetworkInterfaces)}`, region);
+                return rcb();
+            }
+
+            if(!describeNetworkInterfaces.data.length) {
+                helpers.addResult(results, 0, 'No AWS ENIs found', region);
+                return rcb();
+            }
+
+            describeNetworkInterfaces.data.forEach(function(eni){
+                var resource = `arn:${awsOrGov}:ec2:${region}:${accountId}:network-interface/${eni.NetworkInterfaceId}`;
+
+                if (eni.Status === 'in-use') {
```

Comment:
```suggestion
                if (eni.Status && eni.Status === 'in-use') {
```

Reply:
Updated.

Reply:
@AkhtarAmir I don't see this updated in the PR